plugins/flytekit-flyin/flytekitplugins/flyin/utils.py

```diff
@@ -0,0 +1,56 @@
+import importlib
+import os
+import sys
+
+from flytekit.core import utils
+from flytekit.core.context_manager import FlyteContextManager
+from flyteidl.core import literals_pb2 as _literals_pb2
+from flytekit.core.type_engine import TypeEngine
+from flytekit.models import literals as _literal_models
+
+
+def load_module_from_path(module_name, path):
+    """
+    Imports a Python module from a specified file path.
+
+    Args:
+        module_name (str): The name you want to assign to the imported module.
+        path (str): The file system path to the Python file (.py) that contains the module you want to import.
+
+    Returns:
+        module: The imported module.
+
+    Raises:
+        ImportError: If the module cannot be loaded from the provided path, an ImportError is raised.
+    """
+    spec = importlib.util.spec_from_file_location(module_name, path)
+    if spec is not None:
+        module = importlib.util.module_from_spec(spec)
+        sys.modules[module_name] = module
+        spec.loader.exec_module(module)
+        return module
+    else:
+        raise ImportError(f"Module at {path} could not be loaded")
```

Comment:
```suggestion
        raise ImportError(f"Module {module_name} at {path} could not be loaded")
```

Reply:
this module_name is "The name you want to assign to the imported module." So actually you can assign any name to the module that you are going to import.

Reply:
ah i see! that makes sense then